src/pal/tools/gen-buildsys-gcc.sh

```diff
@@ -183,6 +174,6 @@ cmake \
   "-DCMAKE_EXPORT_COMPILE_COMMANDS=1 " \
   "-DCLR_CMAKE_ENABLE_CODE_COVERAGE=$code_coverage" \
   "-DCLR_CMAKE_COMPILER=GNU" \
-  $cmake_extra_defines \
-  $__UnprocessedCMakeArgs \
+  "$cmake_extra_defines" \
```

Comment:
Please remove the quotes here, once restored my cross compilation started working too. Without this change, cross/toolchain.cmake is not getting called during cmake.